SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -101,6 +101,30 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
           appCenter.trackEvent(eventName)
         }
       }
+      for targetToken in TransmissionTargets.shared.transmissionTargets.keys {
+        if TransmissionTargets.shared.targetShouldSendAnalyticsEvents(targetToken: targetToken) {
```

Comment:
You can reverse this `if` for better readability. Use `continue`.

Reply:
@clpolet and I discussed this and it seems options are to either have crazy nested ifs or use `if !TransmissionTargets.shared.targetShouldSendAnalyticsEvents(targetToken: targetToken){` which I don't like because of the `!`, we favor leaving this as is.